build.py

```diff
@@ -126,10 +126,23 @@ def unit_test():
     """
     runid = str(uuid.uuid1())
     python_version = platform.python_version()
-    utility.exec_command(
-        'pytest -l --cov mssqlcli --doctest-modules --junitxml=junit/test-{}-results.xml \
-            --cov-report=xml --cov-report=html --cov-append '
-        '-o junit_suite_name=pytest-{} '
+    utility.exec_command(('pytest -l --cov mssqlcli --doctest-modules '
+                          '--junitxml=junit/test-{}-results.xml --cov-report=xml '
+                          '--cov-report=html --cov-append -o junit_suite_name=pytest-{} '
+                          '-s -m "not unstable" {}').format(runid, python_version,
+                                                            get_active_test_filepaths()),
+                         utility.ROOT_DIR, continue_on_error=False)
+
+def unstable_unit_test():
+    """
+    Run all unstable unit tests.
+    """
+    utility.exec_command('pytest -s -v -m unstable {}'.format(get_active_test_filepaths()),
+                         utility.ROOT_DIR, continue_on_error=False)
```

Comment:
The new function call for unstable tests is much more concise than the stable call because it's not concerned with code coverage.

As an aside, I added a `-v` argument to make test logs more verbose since there are fewer unstable tests.